website/content/docs/connect/config-entries/api-gateway.mdx

```diff
@@ -329,3 +397,166 @@ Specifies the Enterprise [admin partition](/consul/docs/enterprise/admin-partiti
 
 - Default: `"default"` in Enterprise
 - Data type: string
+
+### `Listeners[].default`
+
+Specifies a block of default configurations to apply to the gateway listener. All routes attached to the listener inherit the default configurations. You can specify override configurations that have precedence over default configurations. Refer to the [`override` block](#listeners-override) for details.  
```

Comment:
You can specify overrides on the gateway or on the HTTPRoute. Not sure if this is a good place to describe that, but thought I'd mention it